app/src/main/res/values/strings.xml

```diff
@@ -20,21 +18,25 @@
     <string name="vibrate_behaviour_enabled_key" translatable="false">vibrate_behaviour_enabled_key</string>
     <string name="privacy_policy_shown_key" translatable="false">privacy_policy_shown_key</string>
     <string name="feature_answer_all_angles">Answer at any angle</string>
-    <string name="decline_holder">Declining calls</string>
-    <string name="answer_holder">Answering calls</string>
+    <string name="decline_holder">Hanging up without answering</string>
+    <string name="answer_holder">Answering</string>
     <string name="answer_all_angles_enabled_key" translatable="false">answer_all_angles_enabled_key</string>
-    <string name="missing_support_magnetometer">Your device lacks a magnetometer, declining or limiting answering to an up-right angle is not supported.</string>
-    <string name="missing_support_android_9">Your device is running a version of Android before 9, declining calls is not supported.</string>
+    <string name="missing_support_magnetometer">Can not take angle into account without a magnetometer.</string>
+    <string name="missing_support_android_9">Install Android 9 or later to be able to hang up without answering.</string>
```

Comment:
I don't know if this makes much sense, it's not like people who are running an old Android version can really install a new one in most cases.

Reply:
I am all about promoting the possibility even if it isn't possible.
People ought to know it should be possible, and this is a simpler and shorter of conveying the original content.

Reply:
Fair enough